### ISSUE TITLE
Package Request: Microsoft To Do

### ISSUE THREAD

**Issue by cryocaustik** [microsoft/winget-pkgs#1744]
# Description of the new feature/enhancement

[Microsoft To Do](https://todo.microsoft.com/tasks/) currently is a great integrated solution for quick reminders that blend in well with the Windows 10 environment.

# Proposed technical implementation details (optional)

To Do is currently only available for installation via the Windows Store, which is not always a viable solution; adding this package would provide a simplified alternative.

**Comment by SoftCreatR** [comment]
I had to close my PR for now, because the download link expired. I'll create a new PR with a static link.

**Comment by denelon** [comment]
This looks like it's blocked by https://github.com/microsoft/winget-cli/issues/117
